src/main/kotlin/org/openrewrite/kotlin/KotlinTypeMapping.kt

```diff
@@ -58,15 +58,17 @@ import org.openrewrite.kotlin.KotlinTypeSignatureBuilder.Companion.convertFileNa
 import org.openrewrite.kotlin.KotlinTypeSignatureBuilder.Companion.convertKotlinFqToJavaFq
 
 @Incubating(since = "0.0")
-class KotlinTypeMapping(typeCache: JavaTypeCache, firSession: FirSession) : JavaTypeMapping<Any> {
+class KotlinTypeMapping(typeCache: JavaTypeCache, firSession: FirSession, firFileSymbol: FirFileSymbol) : JavaTypeMapping<Any> {
```

Comment:
The ownerStack will be wrong without access to the file when a `SimpleFunction` is on the top of the stack for the parameters in a top-level method declaration.